core/helpers.nix

```diff
@@ -1,25 +1,57 @@
 # override to the standard nixpkgs lib, adding
 # terranix-related helpers
 pkgs: self: super: {
-  tf = {
+  tf = rec {
     # escape ${} to create a terraform reference string
     ref = ref: "\${${ref}}";
 
     # easier way to create templatefile() invocations
     # if used to set a variable, remember to wrap in `ref`
     template =
-      { text ? ""
-      , source ? ""
-      , variables ? { }
+      {
+        text ? "",
+        source ? "",
+        variables ? { },
```

Comment:
This reformatting doesn't contribute to the purpose of the pull request. A downside of formatting for formatting's sake is that it clutters the version history:
- the most recent history of reformatted lines are no longer tied to the logical changes that introduced them
- the logical changes of this pull request are harder to read

Since you picked nixfmt-rfc-style, it could lead towards having uniform formatting. But I think it's better to deal with this in a separate pull request.